src/main/java/com/mercadolibre/grupo1/projetointegrador/dtos/BatchStock.java

```diff
@@ -0,0 +1,35 @@
+package com.mercadolibre.grupo1.projetointegrador.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+/**
+ * @author Nayara Coca
+ * Classe responsável por filtra dados do estoque no armazém
+ */
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class BatchStock {
```

Comment:
Legal a gente seguir o padrão de por sufixo DTO para os dtos, evita conflitos de nomes entre entidades!
```suggestion
public class BatchStockDTO {
```